### PR TITLE
Update the M2 xml to fulfill the software.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: sal_interfaces/MTM2/MTM2_Commands.xml [hunk 1]
```diff
@@ -1,133 +1,103 @@
...
+        <Units>micron</Units>
+        <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>yTilt</EFDB_Name>
-      <Description>Tilt y in xy plane.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>radian</Units>
-      <Count>1</Count>
+        <EFDB_Name>xTilt</EFDB_Name>
```
Comment (pkubanek) [inline sal_interfaces/MTM2/MTM2_Commands.xml]: I would rather call this rotX, rotY and rotZ (or xRot,..). Tip/tilt are a bit misleading terms. LTS-136 speaks about rotations - no mention of tilt (or tip).

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 2]
```diff
@@ -1,607 +1,275 @@
...
+    </item>
+    <item>
+        <EFDB_Name>xTilt</EFDB_Name>
+        <Description>Rotation about x.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>arcsec</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>yTilt</EFDB_Name>
```
Comment (pkubanek) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: Again either yRot or rotY..

Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 6af5a7b. Thanks!

File: sal_interfaces/MTM2/MTM2_Commands.xml (hunk 1 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Commands.xml]: Fixed in commit: 6af5a7b. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 3]
```diff
@@ -1,607 +1,329 @@
...
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_axialActuatorAbsolutePositionSteps</EFDB_Topic>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: What about we shorten this to MTM2_axialActuatorSteps?
Even if the long topic name doesn't create a problem for EFD, it is not convenient to use.

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 4]
```diff
@@ -1,607 +1,329 @@
...
+    <Description>Axial actuator stepper motor position in steps. Increasing step corresponds to counterclockwise rotation of the motor shaft, looking along the shaft in the direction of the motor. The elements are the actuator Ids from 1 to 72. The related ring Ids are B1-B30, C1-C24, and D1-D18.</Description>
+    <item>
+        <EFDB_Name>axialAbsolutePositionSteps</EFDB_Name>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> axialActuatorSteps

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 5]
```diff
@@ -1,607 +1,329 @@
...
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>double</IDL_Type>
-          <Units>micron</Units>
-          <Count>72</Count>
-      </item>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_tangentActuatorAbsolutePositionSteps</EFDB_Topic>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> MTM2_tangentActuatorSteps

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 6]
```diff
@@ -1,607 +1,329 @@
...
+    <Description>Tangent actuator stepper motor position in steps. Increasing sign indicates extension towards the mirror. The elements are the actuator Ids from 73 to 78. The related ring Ids are A1-A6. The related degrees are 0, 60, 120, 180, 240, and 300.</Description>
+    <item>
+        <EFDB_Name>tangentAbsolutePositionSteps</EFDB_Name>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> tangentActuatorSteps

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 7]
```diff
@@ -1,607 +1,329 @@
...
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>8</Size_in_bytes>
-          <IDL_Type>double</IDL_Type>
-          <Units>micron</Units>
-          <Count>1</Count>
-      </item>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_axialActuatorPositionAbsoluteEncoderPositionsMeasured</EFDB_Topic>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> MTM2_axialEncoderPositions

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 8]
```diff
@@ -1,607 +1,329 @@
...
+    <Description>Axial actuator encoder position. The sign of the position is consistent with the optical coordinate system as per LTS-136. The elements are the actuator Ids from 1 to 72. The related ring Ids are B1-B30, C1-C24, and D1-D18.</Description>
+    <item>
+        <EFDB_Name>axialEncoderPositionsMeasured</EFDB_Name>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> axialEncoderPositions

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 9]
```diff
@@ -1,607 +1,329 @@
...
-          <Frequency>1.0</Frequency>
-          <Publishers>1</Publishers>
-          <Values_per_Publisher>1</Values_per_Publisher>
-          <Size_in_bytes>4</Size_in_bytes>
-          <IDL_Type>unsigned long</IDL_Type>
-          <Units>unitless</Units>
-          <Count>16</Count>
-      </item>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_tangentActuatorPositionAbsoluteEncoderPositionsMeasured</EFDB_Topic>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> MTM2_tangentEncoderPositions

File: sal_interfaces/MTM2/MTM2_Telemetry.xml [hunk 10]
```diff
@@ -1,607 +1,329 @@
...
+    <Description>Tangent actuator encoder position. The sign of the position is consistent with the optical coordinate system as per LTS-136. The elements are the actuator Ids from 73 to 78. The related ring Ids are A1-A6. The related degrees are 0, 60, 120, 180, 240, and 300.</Description>
+    <item>
+        <EFDB_Name>tangentEncoderPositionsMeasured</EFDB_Name>
```
Comment (bxin) [inline sal_interfaces/MTM2/MTM2_Telemetry.xml]: --> tangentEncoderPositions

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 3 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 4 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 5 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 6 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 7 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 8 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 9 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!

File: sal_interfaces/MTM2/MTM2_Telemetry.xml (hunk 10 above)
Comment (teweitsai) [reply sal_interfaces/MTM2/MTM2_Telemetry.xml]: Fixed in commit: 9bb8760. Thanks!
